.github/workflows/release-periodically.yml

```diff
@@ -24,9 +24,24 @@ permissions:
   contents: write
 
 jobs:
+  check-date:
+    # run only on even weeks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if the week number is even
+        id: check
+        run: |
+          week_no=$(date +"%V")
+          if [ $((10#${week_no}%2)) -eq 0 ]; then
```

Comment:
Btw. this is the osbuild version, for inspiration, https://github.com/osbuild/osbuild/blob/main/.github/workflows/create-tag.yml#L15. However, it doesn't look much better.